app/controllers/api/v1/bookmarks_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::BookmarksController < ApplicationController
+  def create
+    bookmark = Bookmark.new(bookmark_params)
+    if bookmark.save
+      render json: BookmarkSerializer.new(bookmark), status: :created
+    else
+      render json: { 'error' => {} }, status: 404
+    end
+  end
+
+  private
+
+  def bookmark_params
+    params.permit(:student_book_id, :date, :minutes, :page_number)
+  end
+end
```

Comment:
Saving the sad paths until later, I like that. I see that there is no show end point, that is within the students controller action bookmarks? Should we specify a bookmark by book?

Reply:
Since we didn't have any bookmarks shown in the wireframes, I figured we didn't need that endpoint yet. However, we can call all bookmarks by student!